src/translate.js

```diff
@@ -688,13 +690,11 @@ const customQuery = ({
 }) => {
   const safeVariableName = safeVar(variableName);
   const [subQuery, subParams] = buildCypherSelection({
-    initial: '',
     cypherParams,
     selections,
     variableName,
     schemaType,
-    resolveInfo,
-    paramIndex: 1
```

Comment:
@Pruxis Is there a reason for removing `paramIndex` here?

Reply:
I think paramIndex defaults to 1, I'm not sure, it's been a month since I did this change

Reply:
Yup, looks like paramIndex default is 1. Thanks